pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -471,13 +489,13 @@ lib.extendDerivation
        args = [ "-c" "export > $out" ];
      });
 
-     inherit meta passthru overrideAttrs;
+     inherit meta passthru env overrideAttrs;
```

Comment:
We should stop exporting implementation details as attributes. A package caller doesn't need to know about `env`, and if they do, it's because either 
 - they need something specific that _should_ be exported. Can be done with `passthru`.
 - they're troubleshooting in the repl. They can use `overrideAttrs` to export `env` temporarily with `passthru`.

Reply:
Makes sense, applied.